### PR TITLE
Encvote fixings

### PR DESCRIPTION
**Pull request by danielSanchezQ**
The `ed25519_dalek::Digest` was causing some problems. The first solution was to use `SHA256`. But Im not sure if this is a problem. Tests run smooth...

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -89,7 +89,7 @@ commands:
         default: ''
       cargo_behavior:
         type: string
-        default: --locked --offline --verbose
```
Comment (mzabaluev) [inline .circleci/config.yml:92]: Drop this for now, we should fix the CI separately and @vincenthz said he has something for this.

File: chain-vote/src/lib.rs [hunk 2]
```diff
@@ -121,6 +123,7 @@ impl Tally {
     }
 }
 
+#[allow(clippy::ptr_arg)]
```
Comment (mzabaluev) [inline chain-vote/src/lib.rs:126]: The last argument could be &[TallyDecryptShare], clippy is right, no need to pass a `Vec` by reference.

**Review by mzabaluev — COMMENTED** [review]
Added some comments, but the PR structure is a bit difficult to review.
Is Vincent (and maybe others) still working on the encvote PR, so this should be incorporated there first?

File: chain-vote/src/lib.rs (hunk 2 above)
Comment (danielSanchezQ) [reply chain-vote/src/lib.rs:126]: I did not feel confident in changing the API. But It makes sense.

**Comment by danielSanchezQ** [comment]
This was fixed already in the main branch. Closing.

**Pull request closed without merging**